insights/specs/default.py

```diff
@@ -915,6 +915,7 @@ def block(broker):
     sshd_config = simple_file("/etc/ssh/sshd_config")
     sshd_config_perms = simple_command("/bin/ls -l /etc/ssh/sshd_config")
     sssd_config = simple_file("/etc/sssd/sssd.conf")
+    dse_ldif = simple_file("/etc/dirsrv/slapd-*/dse.ldif")
```

Comment:
Since you are using a `*` glob char here this should be a `glob_file` instead of `simple_file`.